addons/website/models/ir_ui_view.py

```diff
@@ -160,7 +160,14 @@ def unlink(self):
                     view.with_context(website_id=website.id).write({'name': view.name})
 
         specific_views = self.env['ir.ui.view']
-        if self and self.pool._init:
+
+        # OpenUpgrade: force a False condition to never enter this code block.
+        # This is needed because the update process could correctly unlink
+        # website-agnostic views, but we don't want the website-specific
+        # equivalents to be removed during the migration.
+        if False and self and self.pool._init:
+        # OpenUpgrade: end of modification
+
```

Comment:
Why no remove this chunk of code instead of doing this "if False"? 🤔

Reply:
It's done on purpose for knowing exactly what have been changed. Removing code you can only know what happens seeing commit diff.

Reply:
Yes, if we were patching Odoo I'd never do it like this, but I think that this way the diff is reduced so much but it gets obvious that this is a HACK for OpenUpgrade only. Just in case a reader happens to pass through here.